stream-chat-android-client/build.gradle

```diff
@@ -106,7 +107,10 @@ dependencies {
     androidTestImplementation Dependencies.junit4
     androidTestImplementation Dependencies.espressoCore
     androidTestImplementation Dependencies.androidxTestJunit
+}
 
-    dokkaHtmlPlugin(Dependencies.dokkasaurus)
-    dokkaHtmlPartialPlugin(Dependencies.dokkasaurus)
+tasks.register('dokkaDokkasaurus', DokkaTask) {
+    dependencies {
+        dokkaPlugin(Dependencies.dokkasaurus)
+    }
```

Comment:
Potential later improvement: make the plugin create these tasks for us 😄

Reply:
It would be very good. We can also ask to add the plugin to the official Dokka repository so it comes out of the box.